packages/api/src/command/medical/patient/delete-patient.ts

```diff
@@ -45,20 +44,18 @@ export const deletePatient = async (
         fhirApi.deleteResource("Patient", patient.id).catch(processAsyncError(deleteContext)),
         cqCommands.patient.remove(patient).catch(processAsyncError(deleteContext)),
       ]);
-
-      //eslint-disable-next-line @typescript-eslint/no-explicit-any
-    } catch (err: any) {
-      capture.error(err, {
+      await patient.destroy();
+    } catch (error) {
```

Comment:
Just moved inside the error handling, it should be there so we can have better visibility if it should ever fail